docs/References.bib

```diff
@@ -25,6 +25,15 @@ @article{Beckwith2011
   year={2011},
 }
 
+@book{hartle2003gravity,
+  title={Gravity: An Introduction to Einstein's General Relativity},
```

Comment:
Put Einstein in {}, to prevent it from being turned into lower case by a bibliography processor.